### DISCUSSION TITLE
Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by Rahal1300** [community/community#120697] [GitHub Education]
### Hi everyone,

Hi,
My student pack recently ran out and I was still at school for my last year so I went to renew it, I put my Certificate of attendance for this school year and I was declined as I had to setup 2factor authentication and set my billing name, so I went and did that and resubmitted with the same Certificate and had an error saying I already used that certificate to apply. What should I do next ?

**Comment by samus-aran** [comment]
Hi @Rahal1300 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
